packages/ui/views/account/home/index.tsx

```diff
@@ -105,6 +107,14 @@ class HomeView extends Component<Props, State> {
     await loadingPayPalRequests.wrap(this.props.getPayPalRequests())
   }
 
+  async componentDidUpdate() {
+    const { initialHomeLoad, friendsLoaded, friends } = this.props.state
+    if(initialHomeLoad && friendsLoaded && !friends.length) {
```

Comment:
So this could become, for example:
```
if (initialHomeLoad) {
  this.props.navigation.navigate(initialHomeLoad)
  this.props.setInitialHomeLoad(null)
}
```